docs/_includes/lightbox-dialog.html

```diff
@@ -117,7 +114,6 @@ <h3 id="lightbox-scrolling">Expanded Lightbox</h3>
             <button class="btn btn--primary dialog-button" data-makeup-for="lightbox-dialog-expanded" type="button">Open Scrolling Lightbox Dialog</button>
             <div aria-labelledby="dialog-title-expanded" aria-modal="true" class="lightbox-dialog lightbox-dialog--mask-fade" hidden id="lightbox-dialog-expanded" role="dialog">
```

Comment:
Do we need this expanded dialog now since we're doing something almost identical with the "large" variation?

Reply:
I think we will keep it for now. 
It is mainly used for swapping from bottom dialog to a expanded using the handle
I will add a deprecation notice on that.